language/Spanish_Chile/strings.po

```diff
@@ -0,0 +1,6575 @@
+
+#: src/frontend/mame/ui/ui.cpp:1582
+#, c-format
+msgid "Laserdisc '%1$s' Horiz Position"
+msgstr "Posición horizontal de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1584
+#, c-format
+msgid "Laserdisc '%1$s' Vert Stretch"
+msgstr "Estiramiento vertical de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1587
+#, c-format
+msgid "Laserdisc '%1$s' Vert Position"
+msgstr "Posición vertical de Laserdisc «%1$s»"
+
+#: src/frontend/mame/ui/ui.cpp:1597
+msgid "Vector Flicker"
+msgstr "Vector de parpadeo"
+
+#: src/frontend/mame/ui/ui.cpp:1598
+msgid "Beam Width Minimum"
+msgstr "Ancho de haz mínimo"
+
+#: src/frontend/mame/ui/ui.cpp:1599
+msgid "Beam Width Maximum"
+msgstr "Ancho de haz máximo"
+
+#: src/frontend/mame/ui/ui.cpp:1600
+msgid "Beam Dot Size"
+msgstr "Tamaño del punto del haz"
+
+#: src/frontend/mame/ui/ui.cpp:1601
+msgid "Beam Intensity Weight"
+msgstr "Peso de intensidad del haz"
+
+#: src/frontend/mame/ui/ui.cpp:1614
+#, c-format
+msgid "Crosshair Scale %1$s"
+msgstr "Escala de punto de mira %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1614 src/frontend/mame/ui/ui.cpp:1616
+msgid "X"
+msgstr "X"
+
+#: src/frontend/mame/ui/ui.cpp:1614 src/frontend/mame/ui/ui.cpp:1616
+msgid "Y"
+msgstr "Y"
+
+#: src/frontend/mame/ui/ui.cpp:1616
+#, c-format
+msgid "Crosshair Offset %1$s"
+msgstr "Desplazamiento de punto de mira %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1644
+#, c-format
+msgid "%1$3ddB"
+msgstr "%1$3ddB"
+
+#: src/frontend/mame/ui/ui.cpp:1687
+#, c-format
+msgid "%1$d%%"
+msgstr "%1$d%%"
+
+#: src/frontend/mame/ui/ui.cpp:1702
+#, c-format
+msgid "%1$3.0f%%"
+msgstr "%1$3.0f%%"
+
+#: src/frontend/mame/ui/ui.cpp:1725 src/frontend/mame/ui/ui.cpp:1745
+#: src/frontend/mame/ui/ui.cpp:1764 src/frontend/mame/ui/ui.cpp:1782
+#: src/frontend/mame/ui/ui.cpp:1801 src/frontend/mame/ui/ui.cpp:1820
+#: src/frontend/mame/ui/ui.cpp:1839 src/frontend/mame/ui/ui.cpp:1858
+#: src/frontend/mame/ui/ui.cpp:1879 src/frontend/mame/ui/ui.cpp:1900
+#: src/frontend/mame/ui/ui.cpp:1921 src/frontend/mame/ui/ui.cpp:1942
+#, c-format
+msgid "%1$.3f"
+msgstr "%1$.3f"
+
+#: src/frontend/mame/ui/ui.cpp:1957 src/frontend/mame/ui/ui.cpp:1972
+#: src/frontend/mame/ui/ui.cpp:1987 src/frontend/mame/ui/ui.cpp:2002
+#: src/frontend/mame/ui/ui.cpp:2017
+#, c-format
+msgid "%1$1.2f"
+msgstr "%1$1.2f"
+
+#: src/frontend/mame/ui/ui.cpp:2033
+#, c-format
+msgid "Crosshair Scale X %1$1.3f"
+msgstr "Escala X de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2033
+#, c-format
+msgid "Crosshair Scale Y %1$1.3f"
+msgstr "Escala Y de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2050
+#, c-format
+msgid "Crosshair Offset X %1$1.3f"
+msgstr "Desplazamiento X de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2050
+#, c-format
+msgid "Crosshair Offset Y %1$1.3f"
+msgstr "Desplazamiento Y de punto de mira %1$1.3f"
+
+#: src/frontend/mame/ui/ui.cpp:2137
+msgid "**Error saving ui.ini**"
+msgstr "**Error guardando «ui.ini»**"
```

Comment:
Please change "guardando" with "al guardar". The verb tense is important, it's saying it had an error when attempting to save a file, your text is implying that the error keeps happening.